virocon/contours.py

```diff
@@ -111,11 +167,39 @@ def __init__(self):
 
     @abstractmethod
     def _compute(self):
+        """
+        Compute the contours coordinates.
+
+        Is automatically called in the __init__. 
+        """
         pass
 
 
 class IFORMContour(Contour):
+    """
+    Contour based on the inverse first-order reliability method.
+
+    This method was proposed by Winterstein et. al (1993) [1]_
+
+    Parameters
```

Comment:
we only describe the parameters supplied to the __init__. Maybe we should also describe the Contour objects attributes, e.g. coordinates.